apps/admin-gui/src/assets/i18n/en.json

```diff
@@ -3104,6 +3104,12 @@
         "USER_DONT_EXIST": {
           "TITLE": "The requested user (by Id or external identity) doesn't exist."
         },
+        "USER_NOT_ALLOWED": {
+          "TITLE": "Access not allowed",
+          "MESSAGE": "Service accounts are not allowed to access this application.",
+          "LOGIN": "Redirect to /login",
+          "SERVICE_LOGIN": "Redirect to /service-access"
```

Comment:
This a maybe a preference thing but this is not very elegant. I would honestly maybe put here "Change account" and redirect to login page only.

Reply:
What should be the correct way when someone logs in from the service-access? Should it redirect to it? Or just to the login page?

Reply:
I guess the best option for user is to redirect to login option that was used before. So for OIDC to /login and BA to /service-access.